test/fixtures/linter/autofix/bigquery/004_templating/after.sql

```diff
@@ -1,89 +1,89 @@
-

```

Comment:
I actually only changed lines 27 and 28. The whole file looks different here. Maybe it was in Windows format previously?